src/OpenTelemetry/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## Unreleased
 
+* New instances of `OpenTelemetryLoggerOptions` will no longer be created during
+  configuration reload(s). This was done to prevent the accumulation of extra
+  background threads created by additional `BatchLogRecordExportProcessor`s
```

Comment:
Remove the wording "background threads" from the changelog and PR title. It is really not about threads https://github.com/open-telemetry/opentelemetry-dotnet/pull/5514#issuecomment-2043533241.

Reply:
Consider something like:

Fixed a bug where excessive objects (e.g. processors, exporters) are created/leaked during configuration reloads.

Reply:
Updated